action.yml

```diff
@@ -32,6 +32,10 @@ inputs:
     type: string
     description: By default, waiting workflows will be canceled. When set to true, waiting workflows will instead be canceled.
     default: 'false'
+  tag-superseeded-workflows-with:
```

Comment:
Could you run `npm run build`? It will update the contents of `dist` and get past the dist check on the PR.